openmc/deplete/coupled_operator.py

```diff
@@ -10,6 +10,8 @@
 
 import copy
 from warnings import warn
+import typing
```

Comment:
Just taking a look at this PR out of interest and I couldn't see where this import used so perhaps we don't need it?
```suggestion
```